TriforkSwiftExtensions/TriforkSwiftExtensions/Array/ArrayExtension.swift

```diff
@@ -42,4 +87,34 @@ public extension Array {
         a.append(newElement)
         return a
     }
+
+    /// Returns the first element of the `Array`.
+    var head: Element? { first }
+    
+    /// Removes the first element and retuns a new `Array` containing the rest of the elements.
+    var tail: [Element] { Array(dropFirst()) }
```

Comment:
this should be a function. computed properties are meant for constant operations (O(1))
this is in the best case O(n).
the rational behind this is many, one is: its super easy to use in loops, which would produce quadratic (or worse) runtime performance + the amount of memory consumption is also a concern. besides this does not "access" an element on the array, it actually creates a whole new array. that is very "unexpected" based on a property (usually a "property" is a "property" of a given thing, not a computation per se).